### DISCUSSION TITLE
[Feature Request]: AirPlay 2 remote, "More to Explore" section is missing.

### DISCUSSION THREAD

**Discussion by TheJazzluvr** [ciderapp/Cider#791] [Feature Request]
### The feature you would like

AirPlay 2 casting should be added so we could control Cider using iPhone Control Center. Also it could cast to nearby HomePods and AirPlay compatible speakers.

Oh and "Browse" tab missing the "More to Explore" section. I mean it's there in writing, but there's no content inside to explore which includes Music Videos section. Please fix this. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by quacksire** [comment]
AirPlay 2 requires HomeKit pairing, which is A LOT of overhead and would be more work than it's worth. It'll probably be a post-release thing, after we're done with the general stuff.

If you are a developer who wants to make a AirPlay 2 plugin, be our guests, we can give you all of our research that we gathered on it.


And for the missing section, we'll look into it.
